app/models/camaleon_cms/custom_field_group.rb

```diff
@@ -114,13 +117,13 @@ def before_validating
   # auto save the default field values
   def auto_save_default_values(field, options)
     class_name = self.object_class.split("_").first
-    if ["Post", "Category", "Plugin", "Theme"].include?(class_name) && self.objectid.present? && options[:default_value].present?
+    if class_name.in?(%w(Post Category Plugin Theme)) && self.objectid.present? && options[:default_value].present?
```

Comment:
Use %w since the string has no interpolation. http://stackoverflow.com/questions/5751831/what-is-the-difference-between-w-and-w-upper-and-lower-case-percent-w-arra